src/app/page.tsx

```diff
@@ -1,317 +1,186 @@
-    division: "masters",
-    numFinalists: number
-  ) => (
-    <LeaderboardTable
-      size="small"
-      loading={isFetchingStandings}
-      columns={generateColumns(songs)}
-      dataSource={scores.filter(
-        ({ isDisqualified }) => !shouldHideDisqualified || !isDisqualified
-      )}
-      rowClassName={(record: Standing, index: number) =>
-        `${index < numFinalists ? `${division}-finalist ` : ""}${
-          record.isDisqualified ? "disqualified " : ""
-        }`
-      }
-      pagination={false}
-      rowKey={"ign"}
-      scroll={{ x: true }}
-    />
-  );
+  const [hideDisqualified, setHideDisqualified] = useState<boolean>(true);
+  // const [serverUnixTimestamp, setServerUnixTimestamp] = useState<number>(0);
+  const { data, isLoading } = useSWR<
+    {
+      qualifiers: Standing[];
+      individualSongStandings: IndividualSongStanding[];
+    },
+    any
+  >('/api/submissions', fetcher);
+
+  // useEffect(() => {
+  //   const currentServerUnixTimestamp = getCurrentTime();
+  //   setServerUnixTimestamp(serverUnixTimestamp);
+  // }, [])
 
   return (
     <>
-      {contextHolder}
       <h1>Leaderboard</h1>
-      <p style={{ fontWeight: "bold" }}>{`Qualifiers begin in: ${
-        qualifiersBeginTimeInMilliseconds === undefined
-          ? "---"
-          : formatDuration(qualifiersBeginTimeInMilliseconds)
-      }`}</p>
-      <div style={{ display: "flex", gap: "8px", alignItems: "center" }}>
-        <div
-          style={{
-            display: "flex",
-            flex: "1",
-            flexDirection: "column",
-            gap: "8px",
-          }}
-        >
-          <div style={{ display: "flex", gap: "8px" }}>
-            <Switch
-              onChange={setShouldHideDisqualified}
-              checked={shouldHideDisqualified}
+      <div className="flex items-center space-x-2">
+        <Switch
+          id="hideDisqualified"
+          name="toggleDisqualified"
+          className="my-5"
+          checked={hideDisqualified}
+          onCheckedChange={setHideDisqualified}
+        />
+        <Label htmlFor="hideDisqualified">Hide Disqualified</Label>
+      </div>
+      {/* <Tabs defaultValue="qualifiers" className="w-full">
+        <TabsList className="grid w-full grid-cols-2">
+          <TabsTrigger value="qualifiers">Qualifiers</TabsTrigger>
+          <TabsTrigger value="individuals">
+            Individual Songs (Qualifiers)
+          </TabsTrigger>
+        </TabsList>
+        <TabsContent value="qualifiers"> */}
+      <Table>
+        <TableHeader>
+          <TableRow>
+            <TableHead rowSpan={2}>#</TableHead>
+            <TableHead rowSpan={2}>IGN</TableHead>
+            {qualifierSongs.map((song, index) => (
+              <TableHead key={`${index}${song.songId}`}>
+                <Image
+                  src={song.jacket}
+                  alt={songDetails[song.songId].title}
+                  className="min-h-[90px] min-w-[90px] w-auto h-auto"
```

Comment:
I realised Camellia's long-ass title is making its jacket appear larger than the other 2.

![image](https://github.com/xantho09/chunithmsg/assets/35865440/d796ade6-897e-49b1-b515-70c9966ef75c)

Reply:
Oh I did not realise that LMAO

Will probably fix after merge